### ISSUE TITLE
Quest: Satyr Hooves

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#72]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Durotar / Sen'jin Village / NPC: Ula'elek / Quest: Satyr Hooves
Link: https://www.wowhead.com/wotlk/quest=1842/satyr-hooves

Description: the voice-over suddenly silences during playback.

Progress: no playback.

Completion: the voice-over suddenly silences during playback.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
